recipes/prima/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.7.1" %}
+
+package:
+  name: prima
+  version: {{ version }}
+
+source:
+  fn: prima-{{ version }}.tar.gz
+  url: https://github.com/libprima/prima/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 52bc192444b4ebd07c82ef940a01cfe072e8ae2813a84523ec0df55353de7774
+  patches:
+    - stop.patch  # [win]
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('prima', max_pin='x.x') }}
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ compiler("c") }}               # [unix]
+    - {{ compiler("fortran") }}         # [unix]
+    # flang does not support F2018, use gfortran
```

Comment:
Pinging @isuruf just out of curiosity. You probably already know about this though :grimacing:

Reply:
in particular, flang does not support procedure pointers that we use in the C interface (i'm not sure its actually part of f2018 though)